spec/features/api_spec.rb

```diff
@@ -1,6 +1,6 @@
 require "spec_helper"
 
-RSpec.describe "Suspend a new project with --api flag", type: :feature, autoclean: false do
```

Comment:
The reason why we are doing this is explained [here](https://github.com/thoughtbot/suspenders/pull/1075/files#r702303397).